src/main/java/com/ucl/imaginethisserver/FigmaComponents/Wireframe.java

```diff
@@ -14,7 +15,8 @@
     private String id;
     @Expose()
```

Comment:
no longer to explicitly state `@Expose` since now all fields are exposed by default.

Reply:
Also make sure to remove `spring.gson.exclude-fields-without-expose-annotation=true` in `application.properties`

Reply:
Done